packages/auth/src/accessToken/routes.ts

```diff
@@ -87,7 +87,7 @@ async function rotateToken(
       access_token: {
         access: result.access.map((a) => accessToBody(a)),
         value: result.value,
-        manage: result.managementId,
+        manage: deps.config.authServerDomain + `/token${result.managementId}`,
```

Comment:
The managementId is a uuid so surely you need a `/` after the token here?

Reply:
Yep that's a typo, thanks for catching that